tools/wptrunner/wptrunner/testrunner.py

```diff
@@ -350,6 +351,7 @@ def run(self):
         may also have a stop flag set by the main thread indicating
         that the manager should shut down the next time the event loop
         spins."""
+        self.recording.set(["testrunner", "startup"])
```

Comment:
Thought: would it be better to be explicit about where 'startup' ends (by adding a `pause`)? I can see the argument that "startup is everything until the first test runs" however).

Reply:
I was definitely considering startup to be everything until the first test runs.